dotcom-rendering/src/components/FootballTableList.tsx

```diff
@@ -0,0 +1,105 @@
+import { css } from '@emotion/react';
+import {
+	from,
+	headlineBold17,
+	space,
+	until,
+} from '@guardian/source/foundations';
+import { Stack } from '@guardian/source/react-components';
+import type { ReactNode } from 'react';
+import type { FootballTableData } from '../footballTables';
+import { palette } from '../palette';
+import { FootballTable } from './FootballTable';
+
+const footballTablesGridStyles = css`
+	display: grid;
+	grid-template-columns: [centre-column-start] repeat(4, 1fr) [centre-column-end];
+	column-gap: 10px;
+	${from.mobileLandscape} {
+		column-gap: 20px;
+	}
+
+	${from.tablet} {
+		grid-template-columns: [centre-column-start] repeat(12, 40px) [centre-column-end];
+	}
+
+	${from.desktop} {
+		grid-template-columns: [centre-column-start] repeat(8, 60px) [centre-column-end];
+	}
+
+	${from.leftCol} {
+		grid-template-columns:
+			[left-column-start] repeat(2, 60px)
+			[left-column-end centre-column-start] repeat(8, 60px)
+			[centre-column-end];
+	}
+
+	${from.wide} {
+		grid-template-columns:
+			[left-column-start] repeat(3, 60px)
+			[left-column-end centre-column-start] repeat(8, 60px)
+			[centre-column-end];
+	}
+`;
+
+const CompetitionName = ({ children }: { children: ReactNode }) => (
+	<h3
+		css={css`
+			${until.leftCol} {
+				display: none;
+			}
+			${from.leftCol} {
```

Comment:
An alternate way to write this could be:

```suggestion
			display: none;

			${from.leftCol} {
				display: block;
```

Just saves you a media query.